101-automation/azuredeploy.json

```diff
@@ -0,0 +1,197 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Workspace name"
+            }
+        },
+        "sku": {
+            "type": "string",
+            "allowedValues": [
+                "pergb2018",
+                "Free",
+                "Standalone",
+                "PerNode",
+                "Standard",
+                "Premium"
+            ],
+            "defaultValue": "pergb2018",
+            "metadata": {
+                "description": "Pricing tier: perGB2018 or legacy tiers (Free, Standalone, PerNode, Standard or Premium), which are not available to all customers."
+            }
+        },
+        "dataRetention": {
+            "type": "int",
+            "defaultValue": 30,
+            "minValue": 7,
+            "maxValue": 730,
+            "metadata": {
+                "description": "Number of days to retain data."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Specifies the location in which to create the workspace."
+            }
+        },
+        "automationAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Automation account name"
+            }
+        },
+        "automationAccountLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "Specify the location in which to create the Automation account."
+            }
+        },
+        "sampleGraphicalRunbookName": {
+            "type": "String",
+            "defaultValue": "AzureAutomationTutorial"
+        },
+        "sampleGraphicalRunbookDescription": {
+            "type": "String",
+            "defaultValue": " An example runbook that gets all the Resource Manager resources by using the Run As account (service principal)."
+        },
+        "sampleGraphicalRunbookContentUri": {
+            "type": "String",
+            "defaultValue": "https://eus2oaasibizamarketprod1.blob.core.windows.net/marketplace-runbooks/AzureAutomationTutorial.graphrunbook"
+        },
+        "samplePowerShellRunbookName": {
+            "type": "String",
+            "defaultValue": "AzureAutomationTutorialScript"
+        },
+        "samplePowerShellRunbookDescription": {
+            "type": "String",
+            "defaultValue": " An example runbook that gets all the Resource Manager resources by using the Run As account (service principal)."
+        },
+        "samplePowerShellRunbookContentUri": {
+            "type": "String",
+            "defaultValue": "https://eus2oaasibizamarketprod1.blob.core.windows.net/marketplace-runbooks/AzureAutomationTutorial.ps1"
```

Comment:
I'm still seeing all the uri's pointing to the marketplace storage - not sure if the commit didn't get pushed or something else, but be sure to get all of them.

Reply:
Noticed that my commits weren't completing successfully due to tip of my current branch behind.  After resolving that it appears the PR is up-to-date with the changes.